src/documentation/pages/components/modal/examples/_base.vue

```diff
@@ -0,0 +1,93 @@
+<template lang="pug">
+  <ComponentExample title="Base" id="base" additionalClasses="-position--static" :tabs="exampleTabs">
```

Comment:
it looks like there is no transition (the modal appears immediately while in the old documentation it takes a bit longer and it shows up from top page)